### ISSUE TITLE
Atom feed detection too strict?

### ISSUE THREAD

**Issue by mahemoff** [feedjira/feedjira#310]
[This feed](http://feeds.feedburner.com/skepticcomeskeptic) includes `xmlns:atom="http://www.w3.org/2005/Atom"`. Should Feedjira be identifying it as Atom on that basis?  Feedjira says "no valid parser", I think because it's [expecting](https://github.com/feedjira/feedjira/blob/faae46e3cdc59ab136c276166b2ed8164f27e987/lib/feedjira/parser/atom.rb#L16) `xmlns="http://www.w3.org/2005/Atom"`.

(The example here also uses `rss`, apparently an error, but even with `feed`, it wouldn't match.)

**Comment by mahemoff** [comment]
Actually I think it's some kind of hybrid feed that is more RSS, as I've been able to parse it as RSS. (I had some code to prevent special-casing Feedburner, and for whatever reason, the usual RSS and Atom don't handle the feed.)

**Comment by mahemoff** [comment]
I see the issue now is an inconsistency: RSS parser's [able_to_parse](https://github.com/feedjira/feedjira/blob/master/lib/feedjira/parser/rss.rb#L17) excludes feedburner explicitly, while Atom parser's [able_to_parse](https://github.com/feedjira/feedjira/blob/master/lib/feedjira/parser/atom.rb#L15) doesn't reference feedburner.

I think they should both ignore feedburner or both reference it. I'd personally opt for both ignoring it as it's still correct that they are able to parse RSS/Atom regardless of feedburner being mentioned. ie I'd suggest updating rss.rb to pattern-match just on <rss or <rdf.

**Comment by jonallured** [comment]
Yeah, I could see that - if you submit a PR I'll review and merge.
